### DISCUSSION TITLE
Allow comments on question

### DISCUSSION THREAD

**Discussion by conartist6** [community/community#12753] [Discussions]
My experience with discussions is that they artificially simplify the process of Q&A. It is very, very rare that an answer directly follows a question! This may be true for complete novices asking questions with very obvious answers, but much more of the time there is a back-and-forth needed to determine what the real problem is, what all the relevant symptoms are, and perhaps what solutions have or have not already been tried.

Thus the biggest weak point in the discussions feature is that answers are vote-based, and thus are not suitable for any actual discussion which would only make sense in chronological order. While discussion may result in an answer, it is not itself an answer, and in fact there is no place for it to occur!

My current workaround is to create an answer with the text "discussion" whose comments I can use to discuss. I feel really stupid doing this.

**Comment by JohnyWS** [comment]
# DISCUSSION THREAD

_This is just meant as a thread to discuss the basis for the answers, as [GitHub doesn't support this (yet! - please upvote this feature request!)](https://github.com/github/feedback/discussions/12753)._

**Reply by JohnyWS** [reply]
I've adopted the same behavior now, so thanks for suggesting that - but really is a weird way to handle the discussion... :-/

**Reply by conartist6** [reply]
I like it, and I like that it brings people here.
